Tests/DataDefinitionTests.cs

```diff
@@ -132,9 +132,9 @@ public void TestGreenSystemGreenGold()
 
             // Add a body with green materials and re-test
             StarSystem starSystem = new StarSystem() { systemname = "testSystem" };
-            starSystem.AddBody(jumponiumBody);
+            starSystem.AddOrUpdateBody(jumponiumBody);
             Assert.IsTrue(starSystem.isgreen, "green system should be green");
-            Assert.IsFalse(starSystem.isgold, "green system should not be gold");
+            Assert.IsFalse(starSystem.isgold, "green system should be gold");
```

Comment:
'not' is correct

Reply:
I'm not sure how that got reverted again. Fixed now.